cmd/flarectl/dns.go

```diff
@@ -90,18 +90,18 @@ func dnsCreateOrUpdate(c *cli.Context) {
 	rr := cloudflare.DNSRecord{
 		Name: name + "." + zone,
 	}
-	records, err := api.DNSRecords(zoneID, rr)
+	records, err := api.FilterDNSRecords(zoneID, 0, rr)
```

Comment:
Why do we set page=0 here? The underlying call seems to coerce 0 to 1, but for clarity we should call page=1, which is what the HTTP API accepts.